testsuite/features/profiles/cloud_aws/Kiwi/POS_Image-JeOS7_head/config.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```

Comment:
Do we miss profiles for 4.3 right?
I guess we need to start distinguishing between Head and 4.3, now that 4.3 is released.

Reply:
I think this is not the PR where we should fix these things.

Reply:
I don't get it, this is the PR to include 4.1 profile, but is not the PR to include 4.3 profile? :/

Reply:
testsuite/features/profiles/[Docker|Kiwi]/ are contains templates.
There is a "Makefile" which generate out of these templates the variants needed for running in Nuremberg, Provo or - now new - in the cloud.

The Makefile copy the templates into the new directories and replace some parts using `sed` to make the necessary changes. I just added a new target in the Makefile for cloud and run "make".

I did not change the template where this come from. To create a 4.3 profile, somebody should create it in the template folder and than just run "make".

Reply:
I see. Ok, can you create a card for this purpose?
We need it to have this PR ported to Manager-4.3

Reply:
Done.
https://github.com/SUSE/spacewalk/issues/19147

I am a bit unsure who will get this. I added it to Planning